src/server.ts

```diff
@@ -33,6 +34,7 @@ class App {
     this.app.use(home)
     this.app.use(partners)
     this.app.use(villages)
+    this.app.use(testimonials)
```

Comment:
nit: I think it'd be better if they're lexicographically sorted, so testimonials should appear before village

Reply:
I think it's better to sort it like this, it's explaining which module comes first.